### ISSUE TITLE
Cannot find module in AWS CentOS

### ISSUE THREAD

**Issue by dkrutsko** [gulpjs/gulp-cli#97]
Hey there, so I'm using Gulp for a project and while it works perfectly on OSX I seem to be getting an error whenever I use Gulp on my AWS CentOS server.

```bash
Node Version: 6.8.0
CLI version 1.2.2
Local version 4.0.0-alpha.2
```

```bash
module.js:471
    throw err;
    ^

Error: Cannot find module './src/init'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/ec2-user/code/homepage/node_modules/gulp-sourcemaps/index.js:4:9)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

**Comment by phated** [comment]
Issue in a dependency - see https://github.com/floridoo/gulp-sourcemaps/issues/243

**Comment by phated** [comment]
We've released a version of vinyl-fs (2.4.4) with gulp-sourcemaps pinned to 1.6.0 (before they started breaking everything).
